back-end/routes/stitchRoutes.js

```diff
@@ -56,22 +56,27 @@ router.get('/title/:stitchId', async (req, res) => {
 });
 
 router.post('/create', async (req, res) => {
-    const { title, username, mood, dance, mix, explore } = req.body
+    const { title, username, mood, dance, mix, explore, imageUrl } = req.body
 
     const user = await prisma.user.findUnique({
         where: {
             username: username
         }
     })
 
+    const totalPreferences = mood + dance + mix;
```

Comment:
What happens if the user has zero for all values? Which according to the UI, is possible.

Reply:
Good point, resolved by checking if totalPreferences is equal to 0. If true I set all weights to 0, and then for recommendations I am returning the normal Spotify recommendations now if all are 0.